rails_application/app/services/application_service.rb

```diff
@@ -0,0 +1,17 @@
+class ApplicationService
```

Comment:
This is the first time service objects are used in this repo so I wanted to keep them as simple as possible. In my implementation service objects should only implement `initialize` and `call` methods and then we can call them like `MyService.call(..)`. This makes it easier to mock them (bc we only need to mock the class method `call`).

Please let me know if you prefer to use some other implementation instead, this is just my proposal.